database/migrations/2021_03_18_100000_add_phone_areas.php

```diff
@@ -0,0 +1,1667 @@
+<?php
+
+declare(strict_types = 1);
+
+use Carbon\Carbon;
+use Illuminate\Database\Migrations\Migration;
+use Tipoff\Addresses\Models\Country;
+use Tipoff\Addresses\Models\State;
+
+class AddPhoneAreas extends Migration {
+
+    public function up() {
+
+        if (class_exists(PhoneArea::class)) {
+            PhoneArea::query()->insertOrIgnore([
+                    [
+                        'code' => 239,
+                        'state_id' => 1,
```

Comment:
We can't use an ID for State (#16). We have to look it up as we did with Country in the States migration to add the data:

- https://github.com/tipoff/addresses/blob/main/database/migrations/2015_01_01_125000_add_states.php

This is because the State ID could be different in the application.

Reply:
Sure no problem. I'll change it.

Reply:
Thank you!